projects/batfish/src/org/batfish/grammar/cisco/CiscoParser.g4

```diff
@@ -530,6 +530,7 @@ l_null
    NO?
    (
       ABSOLUTE_TIMEOUT
```

Comment:
Both of these added keywords need to be removed from null_block_substanza or unpredictable behavior will occur.